test/ctia/entity/web_test.clj

```diff
@@ -456,6 +457,10 @@
                (is (= 401 (:status (get-judgement jwt-2))))
                (is (= 2 @counter)
                    "Making a call to another JWT should generate another call")
+               (assert (= test-cache-ttl (p/get-in-global-properties
+                                           [:ctia :http :jwt :http-check :cache-ttl]))
+                       (p/get-in-global-properties
+                         [:ctia :http :jwt :http-check :cache-ttl]))
```

Comment:
Sanity check for an observed flaky test. The `(is (= 3 @counter) ..)` failed for some reason.